lib/email/receiver.rb

```diff
@@ -1329,6 +1329,8 @@ def create_post(options = {})
 
       if result.post
         @incoming_email.update_columns(topic_id: result.post.topic_id, post_id: result.post.id)
+        result.post.update(outbound_message_id: @incoming_email.message_id)
```

Comment:
Just wondering if we need to put the two queries in the same transaction.

Reply:
Yes I think that's a good idea